### ISSUE TITLE
Imposible to remove saved video

### ISSUE THREAD

**Issue by dimqua** [FreeTubeApp/FreeTube#46]
I can't delete saved video from the list. I can only remove all of them. Version 0.1.3-beta.

**Comment by PrestonN** [comment]
I've just confirmed this bug.  I will make sure it is fixed before the next release.  Thanks for the report.

**Comment by sirthorfinn** [comment]
Looking into this. Seems like the app thinks you are on the history page when you try to remove a video and not the saved videos page. Thus tries to call,

```js
deleteHtml = '<i onclick="removeFromHistory(\'' + videoId + '\'); showHistory()" class="videoDelete fas fa-times"></i>';
```

which does not exist in the history .db and nothing happens.

**Comment by PrestonN** [comment]
Fixed as of #54. Closing issue.
